src/Entity.php

```diff
@@ -1044,6 +1092,112 @@ public static function addMultiple(Array $list)
 	}
 
 	/**
+	* Getter/setter for properties of a junction between this and another entity
+	* @access public
+	* @param string $field The field that joins the two entity
+	* @param Entity $otherEntity The other entity to join
+	* @param array $properties Properties to set
+	* @return array The list of properties
```

Comment:
Dans le cas du getter, retourner `$properties` est plutôt logique, mais dans le cas du setter, il faudrait retourner `$this`.